### PR TITLE
PRO-1585 proper client documentation in main docs site

### PR DESCRIPTION
**Pull request by tkfu**

### CODE REVIEW

File: docs/_posts/2016-10-10-client-startup-and-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+----
+[network]
+http_server = "127.0.0.1:8080" <1>
+socket_commands_path = "/tmp/sota-commands.socket" <2>
+socket_events_path = "/tmp/sota-events.socket" <3>
+websocket_server = "https://sota-core.gw.prod01.advancedtelematic.com" <4>
+rvi_edge_server = "127.0.0.1:9080" <5>
+----
+<1> The path to the http-only core server, if the http gateway is enabled in the [gateway] section.
```
Comment (jerrytrieu) [inline docs/_posts/2016-10-10-client-startup-and-configuration.adoc]: This is local host:port to listen for commands over http. Not the sota 'core'.

File: docs/_posts/2016-10-10-client-startup-and-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+<2> The name of the unix domain socket to be used for sending commands, if the socket gateway is enabled in the [gateway] section.
+<3> The name of the unix domain socket to be used for sending events, if the socket gateway is enabled in the [gateway] section.
+<4> The location of the websocket server, for communication with the ATS Garage Core server.
```
Comment (jerrytrieu) [inline docs/_posts/2016-10-10-client-startup-and-configuration.adoc]: This is local host:port to listen for websocket connections from HMI UI.

File: docs/_posts/2016-10-10-client-startup-and-configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+<5> The location of the RVI edge node, if the rvi gateway is enabled in the [gateway] section.
```
Comment (jerrytrieu) [inline docs/_posts/2016-10-10-client-startup-and-configuration.adoc]: This is local host:port to listen for RVI messages from the RVI node.

File: docs/_posts/2016-10-10-client-startup-and-configuration.adoc (hunk 1 above)
Comment (jerrytrieu) [inline docs/_posts/2016-10-10-client-startup-and-configuration.adoc]: Do we need to update this on the client repo?

Comment (tkfu) [inline docs/_posts/2016-10-10-client-startup-and-configuration.adoc]: Oops, that's an oversight on my part. I'll get it fixed. As for updating in the client repo, the idea was to consolidate the docs in one place—so I was going to make a PR removing the redundant info from the client repo once this one is approved.

**Pull request merged**